test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/SignedPackageVerifierTests.cs

```diff
@@ -52,7 +52,7 @@ public SignedPackageVerifierTests(SigningTestFixture fixture)
             using (var testCertificate = new X509Certificate2(_trustedTestCert.Source.Cert))
```

Comment:
@mishra14 should I repeat these same tests for each of the default `SignedPackageVerifierSettings`? or how should I do it to test that the different settings warn or error out in the different scenarios?